<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>940</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="MenuBarWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>940</width>
      <height>71</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
background-color:rgb(56,77,91);
}</string>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>0</y>
       <width>781</width>
       <height>71</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="MenuBar">
      <property name="spacing">
       <number>14</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="SearchArea">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>50</number>
        </property>
        <property name="rightMargin">
         <number>50</number>
        </property>
        <item>
         <widget class="QLineEdit" name="SearchBar">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="LoginBtn">
        <property name="styleSheet">
         <string notr="true">QPushButton{
background-color:rgb(56,77,91);
border:none;
color: rgb(255, 255, 255);
}

QPushButton::hover{
color: rgb(250, 207, 125);
}</string>
        </property>
        <property name="text">
         <string>Login/Registration</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="CartBtn">
        <property name="styleSheet">
         <string notr="true">QPushButton{
background-color:rgb(56,77,91);
border:none;
color: rgb(255, 255, 255);
}

QPushButton::hover{
color: rgb(250, 207, 125);
}</string>
        </property>
        <property name="text">
         <string>Cart</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ForumBtn">
        <property name="styleSheet">
         <string notr="true">QPushButton{
background-color:rgb(56,77,91);
border:none;
color: rgb(255, 255, 255);
}

QPushButton::hover{
color: rgb(250, 207, 125);
}</string>
        </property>
        <property name="text">
         <string>Forum</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="CompanyLogo">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>131</width>
       <height>75</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(252, 233, 79);</string>
     </property>
     <property name="text">
      <string>Spoiled Egg</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QWidget" name="MenuBarWidget_2" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>940</width>
       <height>71</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget{
background-color:rgb(56,77,91);
}</string>
     </property>
     <widget class="QWidget" name="horizontalLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>0</y>
        <width>781</width>
        <height>71</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="MenuBar_2">
       <property name="spacing">
        <number>14</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="SearchArea_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>50</number>
         </property>
         <property name="rightMargin">
          <number>50</number>
         </property>
         <item>
          <widget class="QLineEdit" name="SearchBar_2">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="LoginBtn_2">
         <property name="styleSheet">
          <string notr="true">QPushButton{
background-color:rgb(56,77,91);
border:none;
color: rgb(255, 255, 255);
}

QPushButton::hover{
color: rgb(250, 207, 125);
}</string>
         </property>
         <property name="text">
          <string>Login/Registration</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="CartBtn_2">
         <property name="styleSheet">
          <string notr="true">QPushButton{
background-color:rgb(56,77,91);
border:none;
color: rgb(255, 255, 255);
}

QPushButton::hover{
color: rgb(250, 207, 125);
}</string>
         </property>
         <property name="text">
          <string>Cart</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ForumBtn_2">
         <property name="styleSheet">
          <string notr="true">QPushButton{
background-color:rgb(56,77,91);
border:none;
color: rgb(255, 255, 255);
}

QPushButton::hover{
color: rgb(250, 207, 125);
}</string>
         </property>
         <property name="text">
          <string>Forum</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="CompanyLogo_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>131</width>
        <height>75</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(252, 233, 79);</string>
      </property>
      <property name="text">
       <string>Spoiled Egg</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QScrollArea" name="scrollAreaThread">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>100</y>
      <width>731</width>
      <height>521</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaThreadContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>715</width>
       <height>1500</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>1500</height>
      </size>
     </property>
     <widget class="QTextEdit" name="Post_title">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>351</width>
        <height>70</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Cantarell'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;this is some post title&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="BackToForumB">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>630</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Exit this post</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
